luqum/head_tail.py

```diff
@@ -18,7 +18,7 @@ def __repr__(self):
         return "TokenValue(%s)" % self.value
 
     def __str__(self):
-        return self.value
+        return str(self.value)
```

Comment:
Hum sorry I think this would be better:

```suggestion
        return str(self.value) if self.value else ""
```